script.js

```diff
@@ -37,19 +37,39 @@
 // Write your solution below
 // ====================================================================
 
-
-
-
-
-
-
-
-
-
-
-
-
-
+// var number3 = prompt("Enter a number");
+// var placing = "";
+// if (number3.length === 1) {
+// 	switch(number3) {
+// 		case "1":
+// 			placing = number3 + "st";
+// 			break;
+// 		case "2":
+// 			placing = number3 + "nd";
+// 			break;
+// 		case "3":
+// 			placing = number3 + "rd";
+// 			break;
+// 		default:
+// 			placing = number3 + "th";
+// 	}
+// } else if (number3.length >= 2) {
+// 	switch(true) {
+// 		case (number3.slice(-2) === "11" || number3.slice(-2) === "12" || number3.slice(-2) === "13"):
+// 			placing = number3 + "th";
+// 			break;
+// 		case (number3.slice(-1) === "1"):
+// 			placing = number3 + "st";
+// 			break;
+// 		case (number3.slice(-1) === "2"):
+// 			placing = number3 + "nd";
+// 			break;
+// 		case (number3.slice(-1) === "3"):
+// 			placing = number3 + "rd";
+// 			break;
+// 	}
+// }
+// console.log(placing);
 
```

Comment:
Good job breaking it up into cases. You've covered nearly everything, except.. when I tried to run it with the number 14, I got an empty string...